released/packages/coq-dpdgraph/coq-dpdgraph.0.6.5/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "yves.bertot@inria.fr"
+license: "LGPL 2.1"
```

Comment:
It would be very useful (but not a blocker) to clarify the license here. It looks like this is `LGPL-2.1-only` rather than `LGPL-2.1-or-later`?
```suggestion
license: "LGPL-2.1-only"
```